yaml/pois.yaml

```diff
@@ -1360,8 +1360,11 @@ filters:
       <<: *output_properties
       kind: cycle_barrier
   - filter:
-      shop: [bakery, books, butcher, car, car_repair, clothes, computer, convenience,
-        fashion, florist, gift, greengrocer, hairdresser, jewelry, mobile_phone, optician, pet]
+      shop: [art, bakery, beauty, books, butcher, car, car_repair, clothes,
```

Comment:
`shop=coffee` and `shop=deli` seem more like `amenity=cafe` and should get that **min_zoom** range.

`shop=variety_store` and `shop=furniture` and `shop=shoes` seems more like `craft: [brewery, carpenter...]` and should get that **min_zoom** range.

I'm fine with the rest being `min_zoom: 17`.

Reply:
> shop=variety_store and shop=furniture and shop=shoes seems more like craft: [brewery, carpenter...] and should get that min_zoom range. Even though this has been around awhile, shop=pet should also be in this set.

I'm making the changes, but `craft` is a bit of a weird thing in OSM, and a better word would probably have been `trade`. I don't see the `craft` values and those `shop` values forming much of a coherent set - some are places a customer goes to buy stuff, others don't interact directly with customers at their locations.